### ISSUE TITLE
ChordProParser - whitespaces on the end of the line

### ISSUE THREAD

**Issue by artutra** [martijnversluis/ChordSheetJS#35]
When chordProParser parser this:
```
C                G              F  C/E Dm          C
Whisper words of wisdom, let it be
```
It is ignoring the whitespaces after the Dm chord and formating like this:
```
[C]Whisper words of [G]wisdom, let it [F]be [C/E] [Dm] [C]
```
Shouldn't it format like this?
```
[C]Whisper words of [G]wisdom, let it [F]be [C/E] [Dm]          [C]
```

**Comment by martijnversluis** [comment]
@artutra Looks like a bug indeed! I am going to check whether it is a problem with the parser or the formatter.

**Comment by martijnversluis** [comment]
@artutra I released version `1.5.4` that fixes this bug. I'm sorry it took me so long to complete the PR.
